commit bf116697c55c7c6a199782252f179f90b34a7072 (refs/changes/56/3956/2)
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   2018-05-16T20:40:35+00:00 (1 year, 5 months ago)
    
    OS-6958 bad comparison introduced with OS-6630
    Reviewed by: John Levon <john.levon@joyent.com>
    Reviewed by: Josh Wilsdon <jwilsdon@joyent.com>
    Approved by: Josh Wilsdon <jwilsdon@joyent.com>

diff --git a/src/vm/sbin/vmadmd.js b/src/vm/sbin/vmadmd.js
index 4f1974e2..4c1f7338 100755
--- a/src/vm/sbin/vmadmd.js
+++ b/src/vm/sbin/vmadmd.js
@@ -1126,7 +1126,7 @@ function updateZoneStatus(ev)
         }
 
         // don't handle transitions other than provisioning for non-kvm/bhyve
-        if (['bhyve', 'kvm'].indexOf(vmobj.brand) !== -1) {
+        if (['bhyve', 'kvm'].indexOf(vmobj.brand) === -1) {
             log.trace('doing nothing for ' + ev.zonename + ' transition '
                 + 'because brand "' + vmobj.brand
                 + '" is not "kvm" or "bhyve"');
